### PR TITLE
Fix git checkout commands, those tags do not exist

### PR DESCRIPTION
**Pull request by leto**
Only these 0.14.x tags exist, I updated the docs to use the most recent:
v0.14.0.0
v0.14.0.1
v0.14.0.2

### CODE REVIEW

**Comment by dEBRUYNE-1** [comment]
v0.14.1.0 is soon to be tagged. Similarly, release-v0.14 branch will be created soon. Therefore, I don't think this PR should be merged.

**Comment by leto** [comment]
@dEBRUYNE-1 that is reasonable, and I figured as much. But also, imagine all the people that try to follow your documentation exactly, and the commands do not work. All that confusion. You should not refer to tags that do not exist yet in your documentation. Wait until they exist.

File: README.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ invokes cmake commands as needed.
 * Change to the root of the source code directory, change to the most recent release branch, and build:
 
         cd monero
-        git checkout release-v0.14
+        git checkout release-v0.14.0.2
```
Comment (moneromooo-monero) [inline README.md]: This one does not exist either :P

Comment (leto) [reply README.md]: release-v0.14 does not exist, and you are right, I tried to correct it incorrectly! Wrong prefix, I will update that

Comment (leto) [reply README.md]: @moneromooo-monero ok, fixed! My humble suggestion, as a Git consultant in a former life, is to document soon-to-be-created tags in dev branches, and wait until those tags actually exist to merge those changes into master branch. That will prevent newbies like me from getting confused and retains the flexibility that devs want.

**Comment by dEBRUYNE-1** [comment]
@leto - Good point. Would you then also be able to make the necessary changes when `release-v0.14` branch is created and v0.14.1.0 has been tagged?

**Comment by leto** [comment]
@dEBRUYNE-1 sure, I can update these docs when the new tag is created

**Comment by dEBRUYNE-1** [comment]
@leto - Can you update the PR?

**Comment by moneromooo-monero** [comment]
These have since been created, so this patch is moot.

**Pull request closed without merging**